### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Screen Reader: Narrator/NVDA is not reading the information regarding progress bar.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3361]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier
 

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #NVDA; #Narrator; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
**MAS Violated:** MAS1.3.1

**NOTE:** Same issue is observed after completing the installation process.

**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Fill all the details and press enter on “Next” button.
5)Verify how screen reader is reading information about the progress bar.

**Actual Result:** 
Narrator/NVDA is not reading the information regarding progress bar.

 **Expected Result:** 
Narrator/NVDA should read  the information regarding progress bar.

 **User Impact:** 
Screen Reader users can not get the information about progress if screen reader is not reading anything about the progress bar.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_3361_Screen Reader issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451504/Bug_3361_Screen.Reader.issue.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The Install new software UI and bottom progress is controlled by eclipse, which is out control of our plugin, so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). To verify this issue, you may enter https://aws.amazon.com/eclipse (Url of AWT Eclipse Toolkits) in `Work with` text box and press enter, and Eclipse will behaves same as this issue.

Besides, NVDA could read the progress bar on focus, here is a gif.

![3361-1](https://user-images.githubusercontent.com/12445236/64939367-589d3280-d893-11e9-840f-a08769d83e1b.gif)

**Comment by Flanker32** [comment]
* Importance of experience: Little, eclipse will auto open next page when progress done, so users could install the software without progress bar notification
* Quality of experience: Little, NVDA could read the progress info on hover
* Alternate solutions: Users could use NVDA instead of Narrator to get the progress info
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by Accessibility56** [comment]
Logged an External Bug: https://youtrack.jetbrains.com/issue/IDEA-222757

**Comment by Accessibility56** [comment]
Logged External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551097

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749795
